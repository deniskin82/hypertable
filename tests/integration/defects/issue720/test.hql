
USE "/";

DROP TABLE IF EXISTS TestIssue720;

CREATE TABLE TestIssue720 (
  dtg, status, callsign, origin, destination, origin_time, 
  destination_time, longitude, latitude, type, altitude, speed, heading
);

LOAD DATA INFILE ROW_KEY_COLUMN=dtg "test.tsv" INTO FILE "test.output";

