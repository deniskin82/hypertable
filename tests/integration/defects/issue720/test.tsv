#dtg	status	callsign	origin	destination	origin_time	destination_time	longitude	latitude	type	altitude	speed	heading
2011-08-23 00:46:25	lL	ABC123	ABC	XYZ	11	117	-87.75	41.79	ABC/Q	\N	9999	9999
2011-08-23 00:46:25	lL	DEF123	ABC	XYZ	8	122	-121.59	38.7	ABC/G	\N	9999	9999
