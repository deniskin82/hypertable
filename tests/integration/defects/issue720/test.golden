column	value
status	lL
callsign	ABC123
origin	ABC
destination	XYZ
origin_time	11
destination_time	117
longitude	-87.75
latitude	41.79
type	ABC/Q
speed	9999
heading	9999
status	lL
callsign	DEF123
origin	ABC
destination	XYZ
origin_time	8
destination_time	122
longitude	-121.59
latitude	38.7
type	ABC/G
speed	9999
heading	9999
